File: Back/Domain/CarLoad.cs
using System;

namespace Domain
{
    public class CarLoad
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public float Amount { get; set; }
        public int MaterialId { get; set; }
        public Material Material { get; set; }
        public int PersonId { get; set; }
        public Person Person { get; set; }
        public int DriverId { get; set; }
        public Driver Driver { get; set; }
        public int DestinationId { get; set; }
        public Destination Destination { get; set; }
    }
}

File: OperationProject/src/app/Models/CarLoad.ts
import {Client} from "./client";
import {Driver} from "./driver";
import {Destination} from "./Destination";
import {Material} from "./Material";

export interface CarLoad {
  id: number;
  date: string;
  amount: number;
  material: Material;
  client: Client;
  driver: Driver;
  destination: Destination

}
